### ISSUE TITLE
Version 1.3.3 works, version 1.4.3 doesn't work for me

### ISSUE THREAD

**Issue by Jacowaco** [mesmotronic/air-ane-fullscreen#28]
Hi, so i changed my ANE (1.3.3) to the last ANE that appears in the build folder (1.4.3) and when my app gets to the "AndroidFullScreen.immersiveMode() || AndroidFullScreen.leanMode();" line, it gets stuck, not sure why cause i can't debug. If i change to the old ANE file, it works well. Any idea?

**Comment by neilrackett** [comment]
Hi Jacowaco:

1. Can you try the latest build from the [release page](https://github.com/mesmotronic/air-ane-fullscreen/releases)?
1. Have you set `AndroidFullScreen.stage`?
1. Are you testing on a device or in the AIR simulator?
1. Which AIR SDK are you using?
1. Have you changed the default target SDK in the Android manifest?

**Comment by Jacowaco** [comment]
Now it works! Thanks
1. Nop, i was using it from the build folder downloaded from the master.zip in gitHub
2. Nop, it seems in the previous version i didn't need it so i had skipped it. I added it and now it works!
3. I'm testing in the device.
4. AIR 21
5. Which should be the default?

This is the only code i'm using from this ANE:
AndroidFullScreen.stage = stage;
AndroidFullScreen.immersiveMode() || AndroidFullScreen.leanMode();

Is it ok?

**Comment by neilrackett** [comment]
Glad it's working.

If it's for a game or full screen application, I'd recommend using the new `fullScreen` method, as `leanMode` isn't interactive:

```actionscript3
AndroidFullScreen.stage = stage;
AndroidFullScreen.fullScreen();
```

This will put the device into the best available full screen mode, i.e. immersive mode for devices that support it and Flash's regular full screen mode for those that don't, which means it will also work on iOS devices if you're targeting those.

If it's for video playback, using `leanMode` in the way you are at the moment is still a good option for older devices.

**Comment by Jacowaco** [comment]
Great! Thanks a lot for your help! :)
Is there an specific android minSdkVersion or targetSdkVersion needed?

**Comment by neilrackett** [comment]
No, the AIR SDK defaults are fine.

**Comment by Jacowaco** [comment]
hi neil, is there a way to listen to a method or something so i can know when it finished making it fullscreen so i start starling there? because in fast phones starling loads and then the fullscreen accomodates but the elements in my screen don't.

**Comment by Jacowaco** [comment]
After trying more things i think my problem is somewhere else.
I got a new Moto G3 with Android 6.0, i call:

AndroidFullScreen.stage = stage;
AndroidFullScreen.fullScreen();

But after the fullscreen comes in, i see a black bar where the android buttons were. If i minimize the app and maximize it again, the bar disappears and the fullscreen is full again.
I colored the flash stage.color = 0xff0000 and the starling stage.color = 0x00ff00, and the bar that appears is still black so i don't really know what that black bar is.
